src/app/examples/progress-bar-example/progress-bar-example.component.tns.html

```diff
@@ -1,12 +1,12 @@
 <ScrollView>
   <StackLayout>
     <kirby-card>
-      <kirby-card-header title="Progress bar" subtitle="Default Example" />
-      <kirby-progress-bar progress="33" />
+      <kirby-card-header title="Progress bar" subtitle="Default Example"></kirby-card-header>
```

Comment:
I would not touch this, as it has nothing to do with the fix. Could you revert it?